pkgs/development/python-modules/milksnake/default.nix

```diff
@@ -1,16 +1,15 @@
-{ lib, python3Packages, buildPythonPackage, fetchurl }:
+{ lib, buildPythonPackage, fetchurl, cffi }:
 
 buildPythonPackage rec {
   pname = "milksnake";
   version = "0.1.1";
-  name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://pypi.python.org/packages/04/12/358c2c7a27f06a71d69e44a4b7f62411524c25e6c7284b5a0f757a9ba068/milksnake-0.1.1.zip";
     sha256 = "12bdyc2kjqpiq8wrbsk7ymcq2xdakri3bsvaqgsyzj33wyzbcwzy";
   };
 
-  buildInputs = with python3Packages; [
+  propagatedBuildInputs = [
```

Comment:
Still, how is this going to work? If it uses cffi it tries to load a library, and for that it first needs to find it. How will it find that library outside build-time? And which library does it need?